spec/models/user_spec.rb

```diff
@@ -7,4 +7,47 @@
   it { should have_many :answers }
   it { should have_many :votes }
   it { should have_many :comments }
+  it { should have_many :providers }
+
```

Comment:
Can you please ask Vadim (our head of Ruby) if he wants you to write Capybara Feature Tests for this task?
Because I would. 
But I understand that it's a Twitter OAuth request, which will probably need to be stubbed using something like VCR or Webmock.
Not sure if you need that complexity at this stage.